doc/man3/OSSL_CMP_CTX_new.pod

```diff
@@ -422,14 +426,14 @@ Note that this gives slightly more freedom than OSSL_CMP_CTX_set1_srvCert(),
 which pins the server to the holder of a particular certificate, while the
 expected sender name will continue to match after updates of the server cert.
 
-OSSL_CMP_CTX_set0_trustedStore() sets the certificate store of type X509_STORE
+OSSL_CMP_CTX_set0_trusted() sets the certificate store of type X509_STORE
```

Comment:
Perhaps add something about the old name being an alias here?

Reply:
Good point - done here and for `OSSL_CMP_CTX_get0_trusted()`.